app/views/widgets/image/_form.slim

```diff
@@ -1,3 +1,4 @@
-=> label_tag 'widgets[image][image_url]', 'Image URL:'
-= text_field_tag 'widgets[image][image_url]'
-= hidden_field_tag 'widgets[image][widget_type]', widget_type_id
+#image_widget_form.form-group
+  => label_tag 'widgets[image][image_url]', 'Image URL:'
+  = text_field_tag 'widgets[image][image_url]', options['image_url']
+  = hidden_field_tag 'widgets[image][widget_type]', WidgetType.find_by(:widget_name => 'image').id
```

Comment:
I generally don't think we want to be putting SQL queries into the view like this if we can avoid it. I think the `widget_type_id` might be available inside the options? Or would that only be on edit?